.github/CONTRIBUTING.md

```diff
@@ -32,15 +32,48 @@ traefik*
 
 - You need `go` v1.8+
 - It is recommended you clone Træfik into a directory like `~/go/src/github.com/containous/traefik` (This is the official golang workspace hierarchy, and will allow dependencies to resolve properly)
-- This will allow your `GOPATH` and `PATH` variable to be set to `~/go` via:
+- Set your `GOPATH` and `PATH` variable to be set to `~/go` via:
```

Comment:
Strictly speaking, you only need to set `PATH` to find `go-bindata`, or call it directly from the known path.

Practically speaking, you're probably right because most people define their `go get`ed binaries in terms of the `GOPATH` variable.

So I'm okay with this. :+1: